src/main/java/core/basesyntax/FileBuilder.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import java.util.Scanner;
+
+public class FileBuilder {
+
+    public static void start() {
+        System.out.println("Hi i am FileBuilder ");
+        boolean answerExist = false;
+        while (!answerExist) {
```

Comment:
```suggestion
        while (true) {
```
спробуй просто видалити змінну `answerExist `